modules/essentials/variables.tf

```diff
@@ -1246,3 +1246,31 @@ variable "startupapicheck_image_tag" {
   type        = any
   default     = null
 }
+
+#################
+# metrics-server
+#################
+variable "metrics_server_enabled" {
+  description = "Enable metrics-server helm charts installation."
+  type        = bool
+  default     = true
+}
+
+variable "metrics_server_helm_config_defaults" {
+  description = "Helm provider default config for Metrics Server."
+  type        = any
+  default = {
```

Comment:
Let's set

- Specific image tag 
- Pod resources

if this is not handled by the chart.

Reply:
* if we don't override, app version is the image version.

* resources - https://github.com/kubernetes-sigs/metrics-server#scaling , should we still set some default ?

<img width="823" alt="image" src="https://user-images.githubusercontent.com/46478191/200218996-3dcbdf59-9315-49de-acb0-e0922f9226b6.png">

Reply:
Ok. Sounds good.